aip/client-libraries/4236.md

```diff
@@ -0,0 +1,33 @@
+---
+id: 4236
+state: reviewing
+created: 2024-05-16
+---
+
+# Version-aware clients
+
+APIs can annotate services with [`google.api.api_version`][]. If
+`google.api.api_version` is specified, version-aware clients **must**
+include the value of `google.api.api_version` in the request to the API.
+
+### Expected Generator and Client Library Behavior
+
+If a service is annotated with `google.api.api_version`, client library
+generators **must** include the version in requests sent to the API via
+the HTTP Header `X-Goog-Api-Version`.
```

Comment:
Personally I liked the previous sentence that said `A client **may** send it as a query parameter instead`. Any reason why it was removed?